js/components/TempTester.js

```diff
@@ -33,7 +36,36 @@ function mapStateToProps(state) {
 }
 
 const mapDispatchToProps = (dispatch) => {
+
+  function dispatchMessageFromRDP(packet) {
+    let {type} = packet;
+    if (type === "consoleAPICall") {
+      let message = prepareMessageInput("ConsoleGeneric", packet);
+      dispatch(messageAdd(message))
```

Comment:
That'd be fine, so `let message = null;` above the switch / if blocks and then `if (message) { dispatch(messageAdd(message)) }`  afterwards